src/deepsparse/loggers/helpers.py

```diff
@@ -142,12 +142,13 @@ def possibly_extract_value(value: Any, remainder: Optional[str] = None) -> Any:
         # check whether sub_remainder contains square brackets
         # and thus needs slicing/indexing e.g. `some_key[0:2, 0]`
         square_brackets, sub_remainder = _check_square_brackets(sub_remainder)
-        if not hasattr(value, sub_remainder):
-            raise ValueError(
-                "Attempting to access an non existing "
-                f"attribute {sub_remainder} of an object {value}"
-            )
-        value = getattr(value, sub_remainder)
+        if sub_remainder:
```

Comment:
adapting for the use case where `target_identifier` is e.g. `engine_inputs[0]`, that means `sub_remainder = ''`